src/materials/mineclonia.lua

```diff
@@ -27,21 +29,21 @@ local materials = {
     clay_lump = "default:clay_lump",
     water_bucket = "mcl_buckets:bucket_water",
     empty_bucket = "mcl_buckets:bucket_empty",
-    dye_dark_grey = "mcl_dyes:dark_grey",
-    dye_black = "mcl_dyes:black",
-    dye_white = "mcl_dyes:white",
-    dye_green = "mcl_dyes:green",
-    dye_red = "mcl_dyes:red",
-    dye_yellow = "mcl_dyes:yellow",
-    dye_brown = "mcl_dyes:brown",
-    dye_blue = "mcl_dyes:blue",
-    dye_violet = "mcl_dyes:violet",
-    dye_grey = "mcl_dyes:grey",
-    dye_dark_green = "mcl_dyes:dark_green",
-    dye_orange = "mcl_dyes:orange",
-    dye_pink = "mcl_dyes:pink",
-    dye_cyan = "mcl_dyes:cyan",
-    dye_magenta = "mcl_dyes:magenta",
+    dye_dark_grey = mcl_dyes and "mcl_dyes:dark_grey" or "mcl_dye:dark_grey",
+    dye_black = mcl_dyes and "mcl_dyes:black" or "mcl_dye:black",
+    dye_white = mcl_dyes and "mcl_dyes:white" or "mcl_dye:white",
+    dye_green = mcl_dyes and "mcl_dyes:green" or "mcl_dye:green",
+    dye_red = mcl_dyes and "mcl_dyes:red" or "mcl_dye:red",
+    dye_yellow = mcl_dyes and "mcl_dyes:yellow" or "mcl_dye:yellow",
+    dye_brown = mcl_dyes and "mcl_dyes:brown" or "mcl_dye:brown",
+    dye_blue = mcl_dyes and "mcl_dyes:blue" or "mcl_dye:blue",
+    dye_violet = mcl_dyes and "mcl_dyes:violet" or "mcl_dye:violet",
+    dye_grey = mcl_dyes and "mcl_dyes:grey" or "mcl_dye:grey",
+    dye_dark_green = mcl_dyes and "mcl_dyes:dark_green" or "mcl_dye:dark_green",
+    dye_orange = mcl_dyes and "mcl_dyes:orange" or "mcl_dye:orange",
+    dye_pink = mcl_dyes and "mcl_dyes:pink" or "mcl_dye:pink",
+    dye_cyan = mcl_dyes and "mcl_dyes:cyan" or "mcl_dye:cyan",
+    dye_magenta = mcl_dyes and "mcl_dyes:magenta" or "mcl_dye:magenta",
```

Comment:
would be more ideal to do something like
```lua
local prefix = mcl_dyes and "mcl_dyes" or "mcl_dye"
dye_black = prefix .. ":black"
...
```
thoughts?

Reply:
I can change it to that if you like, though if it's fine enough how it is I won't bother.

Reply:
like buckaroo https://github.com/mt-mods/xcompat/pull/51#issuecomment-2727384195 i dont really care enough. and yeah, prefix is to generic, hence something like